Dockerfile

```diff
@@ -0,0 +1,8 @@
+# syntax=docker/dockerfile:1
+
+FROM node:18-alpine3.19
+# WORKDIR /app
+COPY . .
```

Comment:
It looks like you're copying over every file including node_modules. You'll want to dockerignore the node_modules directory so you're not copying those over.

Reply:
and Done!